test/tools/libtesteth/TestHelper.cpp

```diff
@@ -231,27 +231,47 @@ bytes importByteArray(string const& _str)
     return fromHex(_str.substr(0, 2) == "0x" ? _str.substr(2) : _str, WhenError::Throw);
 }
 
+bytes processDataOrCode(json_spirit::mObject const& _o, string const& nodeName)
+{
+    bytes ret;
+    if (_o.count(nodeName) == 0)
+        return bytes();
+    if (_o.at(nodeName).type() == json_spirit::str_type)
+        if (_o.at(nodeName).get_str().find("0x") != 0)
+            ret = fromHex(replaceBytecode(_o.at(nodeName).get_str()));
+        else
+            ret = importByteArray(_o.at(nodeName).get_str());
+    else if (_o.at(nodeName).type() == json_spirit::array_type)
+    {
+        for (auto const& j : _o.at(nodeName).get_array())
+            ret.push_back(toByte(j));
+    }
+    return ret;
+}
+
 bytes importData(json_spirit::mObject const& _o)
 {
-    bytes data;
-    if (_o.at("data").type() == json_spirit::str_type)
-        data = importByteArray(replaceLLL(_o.at("data").get_str()));
-    else
-        for (auto const& j : _o.at("data").get_array())
-            data.push_back(toByte(j));
-    return data;
+    return processDataOrCode(_o, "data");
 }
 
-string replaceLLL(string const& _code)
+string replaceBytecode(string const& _code)
 {
+    if (_code == "")
+        return "0x";
+    if (_code.substr(0, 2) == "0x" && _code.size() >= 2)
+    {
+        checkHexHasEvenLength(_code);
+        return _code;
+    }
+
     string compiledCode = compileLLL(_code);
     if (_code.size() > 0)
         BOOST_REQUIRE_MESSAGE(compiledCode.size() > 0,
             "Bytecode is missing! '" + _code + "' " + TestOutputHelper::get().testName());
     return compiledCode;
 }
 
-void replaceLLLinState(json_spirit::mObject& _o)
+void replaceBytecodeinState(json_spirit::mObject& _o)
```

Comment:
compileCodeInState

Reply:
Okay will rename `bytecode` to `code` in both functions.